src/main/java/net/mcreator/ui/modgui/ItemGUI.java

```diff
@@ -137,6 +138,9 @@ public ItemGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 		onEntitySwing = new ProcedureSelector(this.withEntry("item/when_entity_swings"), mcreator,
 				"When entity swings item" ,
 				Dependency.fromString("x:number/y:number/z:number/world:world/entity:entity/itemstack:itemstack"));
+		onDroppedByPlayer = new ProcedureSelector(this.withEntry("item/on_dropped"), mcreator,

```

Comment:
Obtain x, y, and z from entity, also world dependency. Check how other procedures that only have entity do this.

Reply:
It's change